.github/dependabot.yml

```
@@ -2,11 +2,11 @@ version: 2
 updates:
 
   - package-ecosystem: "github-actions"
-    directory: "/"
+    directory: "**/*"
     schedule:
       interval: "weekly"
     labels:
-      - "maintenance"
+      - "Maintenance"
```

```suggestion
      - "maintenance"
```
This needs to be modified according to the latest changes in the [labels.yml](https://github.com/ansys/pymapdl/blob/main/.github/labels.yml#L89) file.

---

Good catch.